target/linux/mpc85xx/files/arch/powerpc/boot/dts/ws-ap3825i.dts

```diff
@@ -0,0 +1,246 @@
+// SPDX-License-Identifier: GPL-2.0-or-later or MIT
+
+/include/ "fsl/p1020si-pre.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	model = "Extreme Networks WS-AP3825i";
+	compatible = "extreme-networks,ws-ap3825i";
+
+	aliases {
+		ethernet0 = &enet0;
+		ethernet1 = &enet2;
+		led-boot = &led_power_green;
+		led-failsafe = &led_power_red;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_red;
+	};
+
+	chosen {
+		bootargs-override = "console=ttyS0,115200";
+	};
+
+	memory {
+		device_type = "memory";
+	};
+
+	led_spi {
+		/* This is currently non-functioning because the spi-gpio
+		driver refuses to register when presented with this node. */
+		compatible = "spi-gpio";
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		sck-gpios = <&gpio0 14 GPIO_ACTIVE_HIGH>;
+		mosi-gpios = <&gpio0 13 GPIO_ACTIVE_HIGH>;
+		num-chipselects = <0>;
+
+		spi_gpio: led_gpio@0 {
+			compatible = "fairchild,74hc595";
+			reg = <0>;
+			gpio-controller;
+			#gpio-cells = <2>;
+			registers-number = <1>;
+			spi-max-frequency = <100000>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wifi1 {
+			gpios = <&spi_gpio 3 GPIO_ACTIVE_HIGH>;
+			label = "ws-ap3825i:green:radio1";
+			linux,default-trigger = "phy0tpt";
+		};
+
+		wifi2 {
+			gpios = <&spi_gpio 2 GPIO_ACTIVE_HIGH>;
+			label = "ws-ap3825i:green:radio2";
+			linux,default-trigger = "phy1tpt";
+		};
+
+		led_power_green: power_green {
+			gpios = <&spi_gpio 0 GPIO_ACTIVE_HIGH>;
+			label = "ws-ap3825i:green:power";
+		};
+
+		led_power_red: power_red {
+			gpios = <&spi_gpio 1 GPIO_ACTIVE_HIGH>;
+			label = "ws-ap3825i:red:power";
+		};
+
+		eth0_red {
+			gpios = <&spi_gpio 6 GPIO_ACTIVE_HIGH>;
+			label = "ws-ap3825i:red:eth0";
+		};
+
+		eth0_green {
+			gpios = <&spi_gpio 4 GPIO_ACTIVE_HIGH>;
+			label = "ws-ap3825i:green:eth0";
+		};
+
+		eth1_red {
+			gpios = <&spi_gpio 7 GPIO_ACTIVE_HIGH>;
+			label = "ws-ap3825i:red:eth1";
+		};
+
+		eth1_green {
+			gpios = <&spi_gpio 5 GPIO_ACTIVE_HIGH>;
+			label = "ws-ap3825i:green:eth1";
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "Reset button";
+			gpios = <&gpio0 8 GPIO_ACTIVE_LOW>;
+			linux,code = <0x198>; /* KEY_RESTART */
```

Comment:
#include <dt-bindings/input/input.h> and set linux,code = <KEY_RESTART>; ?

(Just a Q: this reset button is on another GPIO. But it does work, right?)

Reply:
Yes, I confirmed originally and reconfirmed today, this is the GPIO for the reset button and that it does act as such with this node on this GPIO line.

Reply:
Fixed in force-push 557192907b9652b21f1cc8890b807edae22aed40.